llvm/lib/Analysis/VectorUtils.cpp

```diff
@@ -882,7 +882,9 @@ llvm::computeMinimumValueSizes(ArrayRef<BasicBlock *> Blocks, DemandedBits &DB,
 
       // If any of M's operands demand more bits than MinBW then M cannot be
       // performed safely in MinBW.
-      if (any_of(MI->operands(), [&DB, MinBW](Use &U) {
+      auto *Call = dyn_cast<CallInst>(MI);
```

Comment:
```suggestion
      auto *Call = dyn_cast<CallBase>(MI);
```

to handle all call-like instructions that way (even though it won't trigger in LV, there may be other users)?